config/default.yml

```
@@ -4015,6 +4015,26 @@ Style/LineEndConcatenation:
   VersionAdded: '0.18'
   VersionChanged: '0.64'
 
+Style/MagicComments:
+  Description: 'Use a consistent style for magic comments.'
+  Enabled: true
+  VersionAdded: '<<next>>'
+  EnforcedStyle: snake
+  SupportedStyles:
+    # `snake` will enforce the magic comment is written
+    # in snake case (words separated by underscores).
+    # Eg: froze_string_literal: true
+    - snake
```

`snake_case` is consistent with `Naming/MethodName`, `Naming/VariableNumber` option names.

```suggestion
    - snake_case
```

- https://docs.rubocop.org/rubocop/1.34/cops_naming.html#namingmethodname
- https://docs.rubocop.org/rubocop/1.34/cops_naming.html#namingvariablenumber

---

Works for me, I'll fix them up.